src/gui/folderwatcher.cpp

```diff
@@ -57,7 +58,7 @@ bool FolderWatcher::pathIsIgnored( const QString& path )
 
 #ifndef OWNCLOUD_TEST
     if (_folder->isFileExcludedAbsolute(path)) {
-        qDebug() << "* Ignoring file" << path;
+        qCDebug(lcFolderWatcher) << "* Ignoring file" << path;
```

Comment:
If the file is excluded you'll see it in the sync log, and I can't really think of other cases where the user would need this to understand the behavior of the client. Let me know if you think of any.